Newsletters: add 277 (2023-11-15)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Wizardsardine field report

Note: this  PR parameterizes the advertisement for the spaces that we began including two newsletters ago.  The new version uses basically the same text but should automatically stop displaying the ad on a particular newsletter the first time the site is built after the spaces is scheduled to begin.  Let me know if anyone disagrees with that behavior.  In accordance with that behavior, this PR also manually deletes the add from the previous two newsletters.

bitschmidty reviewed (commented):
Concept ACK on the disappearing advert

harding commented on _includes/snippets/recap-ad.md:
Will be pushing an edit that allows translations to be added to the same includes file and will select the correct translation based on the `{{page.lang}}`.  English will be used as a fallback.

harding commented:
Made edits for all feedback (thanks @bitschmidty @LarryRuane !), added lede and releases/rcs, added topic entries, and re-reviewed @darosior section (thanks!).  Thanks everyone!

vostrnad commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+lang: en
+---
+This week's newsletter describes an update to the proposal for ephemeral
+anchors and provides a contributed field report about miniscript from a
```
I've always seen "Miniscript" being capitalized, as pretty much all programming languages are:
```suggestion
anchors and provides a contributed field report about Miniscript from a
```
This would obviously need to be changed throughout the contrib post.

bitschmidty replied:
We tend to lowercase miniscript in the newsletter. 

Although its fair to say maybe I trampled on Antoines writeup with our Optech style preferences :)

vostrnad commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+  (which normally happens during node shutdown, but can also be
+  triggered by the `savemempool` RPC).  Previously, it was stored in a
+  simple serialization of the underlying data.  Now, that serialized
+  structure is xor'd by a random value generated independently by each
```
```suggestion
  structure is XOR'd by a random value generated independently by each
```

vostrnad commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+  triggered by the `savemempool` RPC).  Previously, it was stored in a
+  simple serialization of the underlying data.  Now, that serialized
+  structure is xor'd by a random value generated independently by each
+  node, obfuscating the data.  It's xor'd by the same value during
```
```suggestion
  node, obfuscating the data.  It's XOR'd by the same value during
```

vostrnad commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+  particular sequence of bytes to appear in the saved mempool data, which
+  might cause programs like virus scanners to flag the saved mempool
+  data as dangerous.  The same method was previously applied to storing
+  the block chain in [PR #6650][bitcoin core #6650].  Any software that
```
```suggestion
  the UTXO set in [PR #6650][bitcoin core #6650].  Any software that
```

vostrnad commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+  data as dangerous.  The same method was previously applied to storing
+  the block chain in [PR #6650][bitcoin core #6650].  Any software that
+  needs to read mempool data from disk should be able to trivially apply
+  the xor operation itself, or it use the configuration setting
```
```suggestion
  the XOR operation itself, or use the configuration setting
```

LarryRuane commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+  needs to read mempool data from disk should be able to trivially apply
+  the xor operation itself, or it use the configuration setting
+  `-persistmempoolv1` to request saving in the unobfuscated format.
+  Note, the backwards-compatibility configuration setting is planned to
```
```suggestion
  Note, the backward-compatibility configuration setting is planned to
```

LarryRuane reviewed (commented):
b54052912f21c4c31adc829389ad3a8044f8f498

bitschmidty commented on _posts/en/newsletters/2023-11-15-newsletter.md:
```diff
@@ -0,0 +1,121 @@
+---
+This week's newsletter describes an update to the proposal for ephemeral
+anchors and provides a contributed field report about miniscript from a
+developer working on Revault.  Also included are our regular sections
```
```suggestion
developer working at Wizardsardine.  Also included are our regular sections
```

bitschmidty commented:
Took suggestions from @vostrnad and @LarryRuane , thanks for the feedback.

Made a small change to the lede (wizardsardine). 

Pulled the recap ad snippet into its own commit. Added snippet localization for our current languages and updated older translations to remove the static ad text.

Squashed. Merged! 🚀
